benchmark/scripts/generate_harness/generate_harness.py

```diff
@@ -12,43 +12,70 @@
 #
 # ===---------------------------------------------------------------------===//
 
-# Generate CMakeLists.txt and utils/main.swift from templates.
+# Generate boilerplate, CMakeLists.txt and utils/main.swift from templates.
 
 from __future__ import print_function
 
-import glob
+import argparse
+import jinja2
```

Comment:
@palimondo wrote:

> I don’t see the way to externally parametrize gyb invocations in this way.

You don't need to parameterize them as you could do all the directory walking right in the gyb file itself.  But if you did need to parameterize them, you could use the "expand" function to expand other .gyb templates inline.  The other source of parameterization is `-D name=value` arguments.